Classes/BITHockeyBaseManager.m

```diff
@@ -47,6 +47,14 @@
 #define __IPHONE_6_1     60100
 #endif
 
+#if !defined (HOCKEYSDK_CONFIGURATION_ReleaseCrashOnlyExtensions)
```

Comment:
if it defined then `BIT_UNUSED` has been defined as well?
Suggestion:
```
#ifndef BIT_UNUSED
#define BIT_UNUSED
...
```

Reply:
Never mind, all usages in the same condition too

Reply:
@bmourat I get why we need to do `BIT_UNUSED` as well as the macro, could you please add comments that explain why this is is necessary?